pyrep/robots/arms/arm.py

```diff
@@ -37,6 +38,56 @@ def __init__(self, count: int, name: str, num_joints: int,
         self._collision_collection = vrep.simGetCollectionHandle(
             '%s_arm%s' % (name, suffix))
 
+    def get_configs_for_tip_pose(self, position: List[float],
+                                 euler: List[float] = None,
+                                 quaternion: List[float] = None,
+                                 ignore_collisions=False,
+                                 trials=300, max_configs=60):
```

Comment:
Could you add in the return type? -> List[List[float]]

Reply:
Oh yeah sorry I missed that, thanks!